### PR TITLE
Fixed some blob batch samples

### PR DESCRIPTION
**Pull request by gapra-msft**
Resolves https://github.com/Azure/azure-sdk-for-java/issues/18809

### CODE REVIEW

File: sdk/storage/azure-storage-blob-batch/src/samples/java/com/azure/storage/blob/batch/ReadmeSamples.java [hunk 1]
```diff
@@ -23,12 +23,12 @@
  * Code samples for the README.md
  */
 public class ReadmeSamples {
-    private BlobServiceClient blobServiceClient = new BlobServiceClientBuilder().buildClient();
+    private BlobServiceClient blobServiceClient = new BlobServiceClientBuilder().endpoint("<YOUR_END_POINT>").sasToken("<YOUR_SAS_TOKEN>").buildClient();
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob-batch/src/samples/java/com/azure/storage/blob/batch/ReadmeSamples.java:26]: Need to regenerate the README codesnippets

**Review by alzimmermsft — APPROVED** [review]

File: sdk/storage/azure-storage-blob-batch/src/samples/java/com/azure/storage/blob/batch/ReadmeSamples.java (hunk 1 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-blob-batch/src/samples/java/com/azure/storage/blob/batch/ReadmeSamples.java:26]: How do I do that?

Comment (gapra-msft) [reply sdk/storage/azure-storage-blob-batch/src/samples/java/com/azure/storage/blob/batch/ReadmeSamples.java:26]: None of this should change the lines of code the following snippets are on so I think its ok right?

**Review by kasobol-msft — APPROVED** [review]

**Comment by gapra-msft** [comment]
Merged the other duplicate PR that was opened for this

**Pull request closed without merging**